.github/workflows/docs.yml

```diff
@@ -70,7 +74,6 @@ jobs:
           aws-region: us-east-1
 
       - name: Copy Files to S3
-        shell: bash
         run: |
           echo "current_tag=$(node .github/workflows/scripts/getTag.js --tags "$(git tag -l)")" >> $GITHUB_ENV
```

Comment:
should this line be removed?

Reply:
Ubuntu default shell is `bash` - seems extraneous...

Reply:
I'm referring to the `echo "current_tag=` line here. It's duplicated from the addition above. 
I think an alternative solution would have also been to remove `>> $GITHUB_ENV` so that it sets the env variable at the shell and then could be referenced on the next lines via `$current_tag` instead of `${{ env.current_tag }}`

Reply:
Oh yea, duplicated line... derp.

As far as the alternative, yea I considered that - but thought that having the current tag available to all steps could be beneficial in the future.